docs/plugins/verifier/notation.md

```diff
@@ -260,3 +260,19 @@ Sample Notation CLI config with timestamping configuration:
     }
 }
 ```
+
+## Certificate Revocation Check (CRL)
+
+Ratify supports validating signing identity (certificate and certificate chain) revocation status using [certificate revocation evaluation](https://github.com/notaryproject/specifications/blob/v1.1.0/specs/trust-store-trust-policy.md#certificate-revocation-evaluation) as per revocation setting in Notation trust policy. The CRL implementation uses Notation libraries and follows the [Notary Project specification](https://github.com/notaryproject/specifications/blob/v1.1.0/specs/trust-store-trust-policy.md#crls). 
+
+## Intro
```

Comment:
Suggestion to remove the "intro" heading. This section seems to provided some context around CRL , we can also add in some links to help customer who wants to leverage CRL.

Reply:
Sure